stage2/06-connection-details/01-run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+echo "Installing connection-details and dependencies"
+
+cp files/Pipfile* "${ROOTFS_DIR}"/
+
+on_chroot << EOF
+pip3 install qrcode
+EOF
```

Comment:
I would suggest adding python3-qrcode to packages instead.

Reply:
Wooaaaah, I've been messing around for ages trying to get pipenv to install system packages as root so we can install securely and verify against a checksum, had no idea it was available via apt!